source/addons-and-dependencies/managing-dependencies.md

```diff
@@ -24,7 +24,7 @@ Your own assets (such as `robots.txt`, `favicon`, custom fonts, etc) should be p
 When you're using dependencies that are not included in an addon,
 you will have to instruct Ember CLI to include your assets in the build.
 This is done using the asset manifest file `ember-cli-build.js`.
-You should only try to import assets located in the `bower_components` and `vendor` folders.
+You should only try to import assets located in the `node_modules` and `vendor` folders.
```

Comment:
`bower_components` still works and is necessary for deps that are not using npm

Reply:
Most things are on npm now, so I tend to use exclusively npm, and pull down one off things into vendor. We could mention that bower still works, but that we recommend against it, perhaps. Even bower itself, when you install stuff, tells you to use `yarn` instead of `bower`.